src/main/java/Duke.java

```diff
@@ -1,10 +1,44 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
 public class Duke {
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String greeting = "Hello! I'm Duke\n" + "What can I do for you?";
+        String goodbye = "Bye. Hope to see you again soon!";
+        System.out.println(greeting);
+
+        Task[] tasklist = new Task[100];
+
+        Scanner scanner = new Scanner(System.in);
+        int flag = 0;
+        int index = 0;
+
+        while (flag == 0) {
+            String userinput = scanner.nextLine();
+            if (userinput.equals("bye")) {
+                System.out.println(goodbye);
+                flag = 1;
+            } else if (userinput.equals("list")){
+                for (int i = 1; i <= index; i++) {
+                    System.out.print(i);
+                    System.out.print(".[" + tasklist[i-1].getStatusIcon() + "] " + tasklist[i-1].description + "\n");
+                }
+            } else if (userinput.startsWith("done")){
+                String IndexOfItem = userinput.substring(5);
+                int ID = Integer.valueOf(IndexOfItem) - 1;
```

Comment:
use descriptive names to differentiate between int type and String type variable
eg. "indexString" & "indexInt"